### PR TITLE
AST-1330 - e2e test case for global typography preset3

### PR DESCRIPTION
**Pull request by bsfsnehal**
### Description

Added e2e test case for global typography preset 3 option under the customizer-->Global-->Typography.
### Screenshots

https://share.getcloudapp.com/mXuPY07Y
### How has this been tested?

Execution :- npm run test:e2e:interactive — tests/e2e/specs/customizer/global/typography/preset3.test.js
### Checklist:
- [x] My code is tested
- [ ] My code passes the PHPCS tests
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: tests/e2e/specs/customizer/global/typography/preset3.test.js [hunk 1]
```diff
@@ -0,0 +1,115 @@
+import {
+	createURL,
+	createNewPost,
+	setPostContent,
+	publishPost,
```
Comment (bsfsnehal) [inline tests/e2e/specs/customizer/global/typography/preset3.test.js:5]: need to change path for publish-post util

File: tests/e2e/specs/customizer/global/typography/preset3.test.js [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+			selector: 'body',
+			property: 'font-size',
+		} ).cssValueToBe(
+			`${ globalTypographyPreset3[ 'font-size-body' ].desktop }${ globalTypographyPreset3[ 'font-size-body' ][ 'desktop-unit' ] }`,
+		);
+
+		await setBrowserViewport( 'medium' );
+		await expect( {
+			selector: 'body',
+			property: 'font-size',
```
Comment (SimranChougule1) [inline tests/e2e/specs/customizer/global/typography/preset3.test.js:70]: add responsiveFontSize() for tablet and mobile

Comment (bsfsnehal) [reply tests/e2e/specs/customizer/global/typography/preset3.test.js:60]: Added, but it shows expected and received  value difference error, hence code updated

File: tests/e2e/specs/customizer/global/typography/preset3.test.js [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+			selector: 'body, button, input, select, textarea, .ast-button, .ast-custom-button',
+			property: 'font-weight',
+		} ).cssValueToBe( `${ globalTypographyPreset3[ 'body-font-weight' ] }` );
+		await expect( {
+			selector: 'body',
+			property: 'text-transform',
+		} ).cssValueToBe( `${ globalTypographyPreset3[ 'body-text-transform' ] }` );
+		await expect( {
+			selector: 'body, button, input, select, textarea, .ast-button, .ast-custom-button',
+			property: 'line-height',
```
Comment (SimranChougule1) [inline tests/e2e/specs/customizer/global/typography/preset3.test.js:48]: add code for line height * font size

Comment (bsfsnehal) [reply tests/e2e/specs/customizer/global/typography/preset3.test.js:48]: @simran Still in progress..,I will finalize the PRs for Global Typo and button, and assign to you for review later

**Review by GauravKhupse — APPROVED** [review]

**Review by SimranChougule1 — APPROVED** [review]

**Pull request merged**